test/integration.js

```diff
@@ -198,8 +198,8 @@ describe('integration tests:', function () {
       });
 
       it('should execute a long command with arguments', function (done) {
-        exec('very complicated deep command abc123 -rad -sleep \'well\' -t -i \'j\' ', done, function () {
-          stdout().should.equal('radtjabc123');
+        exec('very complicated deep command abc123 -rad --sleep \'well\' -t -i \'j\' ', done, function () {
```

Comment:
Uh, sleep shouldn't be able to accept an argument as per its declaration:

https://github.com/dthree/vorpal/blob/master/test/util/server.js#L211